self-healing/transform-ldes-data-to-entities.ts

```diff
@@ -101,15 +110,66 @@ export async function transformRemainingEntities() {
       }
     }
     WHERE {
+      { SELECT ?ldesEntity WHERE {
+          GRAPH ${sparqlEscapeUri(HEALING_DUMP_GRAPH)} {
+            ?ldesStream tree:member ?ldesEntity.
+            ?ldesEntity prov:generatedAtTime ?time.
+          }
+      } LIMIT 10000 }
```

Comment:
Do we want this to be configurable?

Reply:
didn't want to confuse by adding yet another env var and felt this was a sane default that any virtuoso should be able to handle